src/AppBundle/Entity/DailyOccurrence.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of Eventbase API.
+ *
+ * (c) 2017–2018 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace AppBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A daily occurrence of an Event.
+ *
+ * @ORM\Entity
+ *
+ * @ApiResource(
+ *   collectionOperations={
+ *     "get"={
+ *       "method"="GET",
+ *       "path"="/calendar",
```

Comment:
Why not `/daily_occurrences`?

Reply:
When we started talking about this it was `/calendar` - now changed to `/daily_occurrences` for consistency